### DISCUSSION TITLE
sFlow data collected, but not accessible in the web

### DISCUSSION THREAD

**Discussion by CollinSchneeweiss** [akvorado/akvorado#807] [Q&A]
Hey,

I have a sFlow agent on a Debian 12 machine and want to export the sFlow data to the exporter for the Akvorado panel. The panel works.

Here is the following output of curl -s http://localhost:8080/api/v0/inlet/metrics | grep '^akvorado_inlet':

akvorado_inlet_core_classifier_exporter_cache_size_items 0
akvorado_inlet_core_classifier_interface_cache_size_items 0
akvorado_inlet_core_flows_errors{error="SNMP cache miss",exporter="10.0.0.41"} 7571
akvorado_inlet_core_flows_errors{error="input and output interfaces missing",exporter="10.0.0.41"} 1134
akvorado_inlet_core_flows_http_clients 0
akvorado_inlet_core_flows_received{exporter="10.0.0.41"} 8705
akvorado_inlet_flow_decoder_count{name="sflow"} 1585
akvorado_inlet_flow_decoder_sflow_count{agent="10.0.0.41",exporter="10.0.0.41",version="5"} 1585
akvorado_inlet_flow_decoder_sflow_sample_records_sum{agent="10.0.0.41",exporter="10.0.0.41",type="CounterSample",version="5"} 6804
akvorado_inlet_flow_decoder_sflow_sample_records_sum{agent="10.0.0.41",exporter="10.0.0.41",type="FlowSample",version="5"} 7571
akvorado_inlet_flow_decoder_sflow_sample_sum{agent="10.0.0.41",exporter="10.0.0.41",type="CounterSample",version="5"} 1134
akvorado_inlet_flow_decoder_sflow_sample_sum{agent="10.0.0.41",exporter="10.0.0.41",type="FlowSample",version="5"} 7571
akvorado_inlet_flow_input_udp_bytes{exporter="10.0.0.41",listener="0.0.0.0:6343",worker="3"} 1.777768e+06
akvorado_inlet_flow_input_udp_in_drops{listener="0.0.0.0:6343",worker="3"} 0
akvorado_inlet_flow_input_udp_packets{exporter="10.0.0.41",listener="0.0.0.0:6343",worker="3"} 1585
akvorado_inlet_flow_input_udp_summary_size_bytes{exporter="10.0.0.41",listener="0.0.0.0:6343",worker="3",quantile="0.5"} 1240
akvorado_inlet_flow_input_udp_summary_size_bytes{exporter="10.0.0.41",listener="0.0.0.0:6343",worker="3",quantile="0.9"} 1372
akvorado_inlet_flow_input_udp_summary_size_bytes{exporter="10.0.0.41",listener="0.0.0.0:6343",worker="3",quantile="0.99"} 1396
akvorado_inlet_flow_input_udp_summary_size_bytes_sum{exporter="10.0.0.41",listener="0.0.0.0:6343",worker="3"} 1.777768e+06
akvorado_inlet_flow_input_udp_summary_size_bytes_count{exporter="10.0.0.41",listener="0.0.0.0:6343",worker="3"} 1585
akvorado_inlet_snmp_cache_expired 0
akvorado_inlet_snmp_cache_hit 0
akvorado_inlet_snmp_cache_miss 7571
akvorado_inlet_snmp_cache_refresh 0
akvorado_inlet_snmp_cache_refresh_runs 4
akvorado_inlet_snmp_cache_size 0
akvorado_inlet_snmp_poller_breaker_open_count{exporter="10.0.0.41"} 1354
akvorado_inlet_snmp_poller_coalesced_count 5991
akvorado_inlet_snmp_poller_error_requests{error="get",exporter="10.0.0.41"} 114
akvorado_inlet_snmp_poller_pending_requests 0
akvorado_inlet_snmp_poller_retry_requests{exporter="10.0.0.41"} 228

Any ideas how to fix it? I am not sure what is exactly meant by 'input and output interfaces missing', should it be the interface in the core config?

**Comment by vincentbernat** [comment]
It looks like SNMP is not configured.

**Reply by CollinSchneeweiss** [reply]
Thanks for the reply! The data is sent directly via UDP to dst port 6343 not SNMP. I also can see the traffic incoming via tcpdump.

**Reply by vincentbernat** [reply]
You need SNMP to collect interface names.

**Reply by agust-d23** [reply]
Hi @vincentbernat , should I enable SNMP on the akvorado server or on the exporter device? waiting for your reply

**Reply by vincentbernat** [reply]
On the exporter device.
